services/web-app/components/mdx-icon/mdx-icon.js

```diff
@@ -51,7 +55,9 @@ const svgIcons = {
   adobe24: Svg24Adobe,
   axure24: Svg24Axure,
   adobe32: Svg32Adobe,
-  axure32: Svg32Axure
+  axure32: Svg32Axure,
+  xd: Svg32AdobeXd,
```

Comment:
should this be `xd32` instead?
```suggestion
  xd32: Svg32AdobeXd,
```